params/config.go

```diff
@@ -378,6 +381,18 @@ func (c *CliqueConfig) String() string {
 	return "clique"
 }
 
+// OptimismConfig is the optimism config.
+type OptimismConfig struct {
+	Enabled          bool           `json:"enabled"`
```

Comment:
The other json config options use camelcase

Reply:
Done.